wiki/Rules/Song_content_rules/vi.md

```diff
@@ -0,0 +1,24 @@
+# Quy định nội dung nhạc
+
+Với nhiều thể loại nhạc khác nhau, hầu hết các bài hát/nhạc cover đều có thể sử dụng trong osu!, nhưng hãy cung cấp các bài nhạc không chứa những nội dung liên quan như :
+
+* Liên quan nặng nề đến vấn đề chính trị
+* Phân biệt hoặc chia rẽ chủng tộc
+* Cực kỳ ồn ào và đến mức không thể chịu được 
+* Đã được sửa đổi (chứa âm thanh đã bị cắt) hoặc móp méo (như chỉnh EQ tránh bản quyền) bài nhạc không theo mong muốn của tác giả
+
+Hãy lưu ý rằng các bài nhạc không phù hợp với trẻ em [`sẽ phải tuân theo các quy định khác nhau`](https://osu.ppy.sh/wiki/vi/Rules/Explicit_content).
+
+## Nội dung nào được phân loại là nội dung "mang tính chính trị nặng nề"?
+
+Vì lợi ích của việc giữ một trò chơi về nhịp điệu chỉ liên quan đến nhịp điệu một cách nghiêm khắc và phù hợp với [`Quy tắc cộng đồng`](https://osu.ppy.sh/wiki/vi/Rules#community-rules)#3, các nội dung nhạc mà chính quyền cho là liên quan đến mặt chính trị một cách thái quá thì không được phép sử dụng.
+
+Chúng tôi không tuân thủ bất kỳ "chủ định" cụ thể nào liên quan đến phạm vi chính trị - osu! là nơi hoàn toàn phi chính trị và chúng tôi muốn mọi người vui vẻ cùng nhau, chứ không phải nơi để gây ra mối thù hằn hay cãi nhau về chính trị.
+
+## Chuyện gì sẽ xảy ra nếu bản nhạc của tôi không ổn hoặc nằm trong diện bị cấm như đã nêu ở trên?
+
+Bất kỳ beatmap nào có chứa nhạc được nêu như trên sẽ bị xóa khỏi website.
+
+Bạn có thể tranh luận với họ qua PM (chat riêng với họ) nếu bạn muốn, hoặc nghe ý kiến của những người khác để chứng minh được sự phù hợp của bài hát đấy. Nếu bạn vẫn không tìm được một điểm chung nào để đối đầu trong việc này, yêu cầu của bạn có thể sẽ bị đưa lên đến việc các thành viên của NAT phải bỏ phiếu nhất trí, để có được một quyết định chung xem được rằng bài nhạc của bạn có được chấp nhận hay không.
+
+Ngoại trừ các bản nhạc hoặc nội dung tải lên không phù hợp đến mức nghiêm trọng, người dùng sẽ bị  **one submission leeway** trước khi hình phạt đưa ra do tính chất không chắc chắn của phán quyết này, sau khi "leeway" đã trôi qua, nếu người dùng vẫn tiếp tục lựa chọn tải lên các bài không phù hợp sẽ đối mặt với một khoảng thời gian [`im lặng`](https://osu.ppy.sh/wiki/vi/Silence) lên đến 42 giờ cùng với việc bài nhạc của họ bị gỡ 
```

Comment:
this might be missing a paragraph, along with a link

![image](https://github.com/ppy/osu-wiki/assets/36758269/2ee612fd-ed18-4123-a5d4-007628311c24)

Reply:
> điều này có thể thiếu một đoạn văn, cùng với một liên kết
> 
> ![hình ảnh](https://private-user-images.githubusercontent.com/36758269/331416441-2ee612fd-ed18-4123-a5d4-007628311c24.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5MDYwMzcsIm5iZiI6MTcxNTkwNTczNywicGF0aCI6Ii8zNjc1ODI2OS8zMzE0MTY0NDEtMmVlNjEyZmQtZWQxOC00MTIzLWE1ZDQtMDA3NjI4MzExYzI0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE3VDAwMjg1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkODNjYzMzMDBjYTYzYjI0MmE2YmQ0OTFjYmZkY2MzNWRjNzEzYTViZjU3ZjZkYTcwYWNiMTIyZDk2NjIwMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.O_ufZkSWtgZkhQymMTrtvLswpySyWZ_1PS6d9TLScLU)

I have added it

Reply:
NAT and GMT line is missing